packages/mesa.rb

```diff
@@ -26,49 +18,34 @@ class Mesa < Package
   depends_on 'elfutils'
   depends_on 'llvm'
 
-  def self.patch
-    system "wget -O- https://patchwork.freedesktop.org/patch/257229/raw | patch -Rp1"
-  end
-
   def self.build
     system "pip3 uninstall -y Mako MarkupSafe || :"
     system "pip3 install --prefix \"#{CREW_PREFIX}\" --root \"#{CREW_DEST_DIR}\" Mako==1.0.7"
     system "pip3 install --prefix \"#{CREW_PREFIX}\" Mako==1.0.7"
```

Comment:
Are we certain Mako should be pinned to 1.0.7 for this release?

Reply:
Indeed : "Python Mako module - Python Mako module is required. Version 0.8.0 or later should work." Going to change that

Reply:
OK, done. I had to add setuptools dependency to build step